spec/models/branch_of_service_contact_spec.rb

```diff
@@ -0,0 +1,5 @@
+require 'rails_helper'
+
+RSpec.describe BranchOfServiceContact, type: :model do
+  pending "add some examples to (or delete) #{__FILE__}"
+end
```

Comment:
Remove this file unless there are specs to write against any custom methods in the model (which there aren't currently).